2024-02-15 post (#28)
SheffieldR/SheffieldR.github.io: posts/2024-02-15/index.qmd

---
layout: post
title:  "SheffieldR : February 2024 Meetup"
date:   2024-02-15 17:30:00 +0000
categories: meetups
image: image.jpg
excerpt_separator: <!--more-->
---

> Apologies for the short notice but we have a volunteer, Simon Rolph, and so have scheduled another talk.

Reviewer: ```suggestion
Apologies for the short notice but thanks to Simon Rolph, who has [volunteered to give a talk](https://docs.google.com/forms/d/e/1FAIpQLSfjbSGzIHs71YCxD_2knDg126q7m0ywFwF7rspqyVpmZolxpg/viewform), we have scheduled a meeting for February.
```

Meeting is 2024-02-15 17:30-19:00 in Hicks Building LT9. Sign up via
[Meetup](https://www.meetup.com/sheffieldr-sheffield-r-users-group/events/299095698/). Remote attendance is possible
using [Google Meet](meet.google.com/ahi-irru-rvx).

If you have any questions or enquiries please email us [sheffieldr@sheffield.ac.uk](mailto:sheffieldr@sheffield.ac.uk).

<!--more-->

> ## Simon Rolph

Reviewer: ```suggestion
## Dive into [{targets}](https://books.ropensci.org/targets)
```

Delve into the dynamic world of the [{targets}](https://books.ropensci.org/targets) R package—a powerful tool designed
to streamline and enhance your data science and analysis workflows.

> Targets is not just another package; it's a game-changer for reproducibility, efficiency, and scalability in your R

Reviewer: Seems that targets shouldn't be capitalised?
```suggestion
targets is not just another package; it's a game-changer for reproducibility, efficiency, and scalability in your R
```

> projects. Developed to manage complex dependency graphs, Targets automates the execution of your analyses, making it

Reviewer: ```suggestion
projects. Developed to manage complex dependency graphs, targets automates the execution of your analyses, making it
```

easier to track changes.

In this session we'll introduce the targets package, discuss why it's useful and explore the core features of the
targets package by defining a basic pipeline.

### Biography - Simon Rolph

> Rob is a Data Scientist at the [UK Centre for Ecology and Hydrology](https://www.ceh.ac.uk/) where he works to enhance

Reviewer: ```suggestion
Simon is a Data Scientist at the [UK Centre for Ecology and Hydrology](https://www.ceh.ac.uk/) where he works to enhance
```
